frontend/src/components/Block.tsx

```diff
@@ -0,0 +1,87 @@
+import React from "react";
```

Comment:
We should use `import * as React` throughout. This is the suggested import method and will be enforced in the future: https://github.com/facebook/react/pull/18102

Reply:
👍